test/fake-api-fixtures/rules/index.json

```diff
@@ -0,0 +1,33 @@
+{
+  "meta": {
+    "name": "cumulus-api",
+    "stack": "test-src-integration",
+    "table": "rule",
+    "limit": 10,
+    "page": 1,
+    "count": 1
+  },
+  "results": [
+    {
+      "name": "MOD09GQ_TEST_kinesisRule",
+      "createdAt": 1538105473704,
+      "workflow": "KinesisTriggerTest",
+      "provider": "PODAAC_SWOT",
+      "meta": {
+        "cnmResponseStream": "test-src-integration-cnmResponseStream"
+      },
+      "rule": {
+        "type": "kinesis",
+        "arn": "12345678-1234-1234-abc1-1a2b3c4d5e6f",
+        "value": "arn:aws:kinesis:us-east-1:12345678:stream/test-src-integration-testStream-12345678-KinesisError"
+      },
+      "collection": {
+        "name": "MOD09GQ",
+        "version": "006"
+      },
+      "state": "ENABLED",
+      "updatedAt": 1538105473704,
+      "timestamp": 1538105475132
+    }
+  ]
+}
```

Comment:
Do you need `rules/MOD09GQ_TEST_kinesisRule/index.json` to view individule rule?

Reply:
Yeah, I didn't add that just because it's not a part of these tests, but I can